components/ResourceTable.tsx

```diff
@@ -1,47 +1,148 @@
-import React, { useState } from "react";
+import React, {useEffect, useState} from "react";
 import Link from "next/link";
 import BrTable from "./brickroom/BrTable";
-import QrCodeButton from "./brickroom/QrCodeButton";
+import {gql, useQuery} from "@apollo/client";
+import devLog from "../lib/devLog";
+import BrLoadMore from "./brickroom/BrLoadMore";
+import Spinner from "./brickroom/Spinner";
 
 const truncate = (input: string, max: number) => input.length > max ? `${input.substring(0, max)}...` : input;
 
+const FETCH_INVENTORY = gql`query($first: Int, $after: ID, $last: Int, $before: ID, $filters:EconomicResourceFilterParams ) {
+  economicResources(first: $first after: $after before: $before last: $last filter: $filters) {
```

Comment:
first -> $first
after -> $after
.
.
.

Then why `filter` becomes `$filters` in plural, why they do not match, like all other fields?